test/workerd/tests.mjs

```diff
@@ -58,3 +58,13 @@ export const buffer_implements = {
     assert.ok(new Buffer.Blob([]));
   },
 };
+
+// --- node:util
+
+export const util_implements = {
+  async test() {
+    const { types } = await import("unenv/runtime/node/util");
+    assert.strictEqual(types.isExternal("hello world"), false);
```

Comment:
Is there a simple test that we can add that the return value is `true` today?

I checked cloudflare/workerd#2264 but no clues and [node docs](https://nodejs.org/api/util.html#utiltypesisexternalvalue) uses node bindings as example. does workerd support napi?

I am asking this mainly because if there are really no real cases, i was thinking the nodeless polyfill (when node_compat is not enabled) could be `false` perhaps?

Reply:
there is no real case. workerd doesn't support napi.

Reply:
then is it safe to assume it always returns `false`?

Reply:
I think so. @jasnell wdyt?

Reply:
If in case we agree (i think you are experts to say this of course!), i would suggest to update [here](https://github.com/unjs/unenv/blob/main/src/runtime/node/util/internal/types.ts#L4) to reflect this and always return `false`.

This way when we enable matrix for nodeless, tests pass.

Reply:
workerd currently does not expose any `External`s to public API that can be tested.